languages/rust/template/algorithmia.conf

```diff
@@ -0,0 +1,5 @@
+{
+    "username": "__USER__",
+    "algoname": "__ALGO__",
+    "language": "rust"
```

Comment:
I think it should be versioned, as even though every release states it's BWC - sometimes there are some minor things (jmalloc?) that aren't and could lead to undefined behavior.

Reply:
Jmalloc was technically backward compatible. But yes, there are exceptions, usually because of compiler soundness bugs, but as someone who has authored 15+ rust projects over 4 years, never once has a stable compiler upgrade broke my old code; but not upgrading has made it so I couldn't add/update deps.

So as long as there is a straight-forward way to upgrade the compiler for an algorithm every quarter or two, then versioning the rust packageset sounds fine to me

Reply:
Upgrading the compiler would be:
1. rebuild the rust package set (assuming it's always pulling the latest) 
2. upgrading the version of the packageset in use for an algorithm to the latest build (if you set `packageSetName` when doing a create/update algorithm API call this will happen - we don't have a way yet to show something in the UI like "by the way you can upgrade your base image!")

Reply:
I interpret that as: I don't have to introduce any sort of version in this algorithmia.conf. Is that correct?